tools/runners/run-mongo.js

```diff
@@ -36,24 +37,33 @@ var runMongoShell = function (url) {
 };
 
 // Start mongod with a dummy replSet and wait for it to listen.
-var spawnMongod = function (mongodPath, port, dbPath, replSetName) {
-  var child_process = require('child_process');
+function spawnMongod(mongodPath, port, dbPath, replSetName) {
+  const child_process = require('child_process');
 
   mongodPath = files.convertToOSPath(mongodPath);
   dbPath = files.convertToOSPath(dbPath);
 
-  return child_process.spawn(mongodPath, [
-      // nb: cli-test.sh and findMongoPids make strong assumptions about the
-      // order of the arguments! Check them before changing any arguments.
-      '--bind_ip', '127.0.0.1',
-      '--smallfiles',
-      '--port', port,
-      '--dbpath', dbPath,
-      // Use an 8MB oplog rather than 256MB. Uses less space on disk and
-      // initializes faster. (Not recommended for production!)
-      '--oplogSize', '8',
-      '--replSet', replSetName
-  ], {
+  const args = [
+    // nb: cli-test.sh and findMongoPids make strong assumptions about the
+    // order of the arguments! Check them before changing any arguments.
+    '--bind_ip', '127.0.0.1',
+    '--port', port,
+    '--dbpath', dbPath,
+    // Use an 8MB oplog rather than 256MB. Uses less space on disk and
+    // initializes faster. (Not recommended for production!)
+    '--oplogSize', '8',
+    '--replSet', replSetName
+  ];
+
+  // Use mmapv1 on windows, as our binary doesn't support WT
+  if (process.platform === "win32") {
+    args.push('--storageEngine', 'mmapv1', '--smallfiles');
+  } else {
+    // The WT journal seems to be at least 300MB, which is just too much
+    args.push('--nojournal');
```

Comment:
I mean we didn't use this option before and I added it specifically to change the behavior of WT, so to me it seems strange to turn it on for mmvap1.